evaluate/src/main.rs

```diff
@@ -57,26 +46,26 @@ impl FromStr for EvaluationMetric {
 )]
 struct Args {
     /// The model file to use when analyzing text
-    #[clap(long)]
+    #[clap(long, action)]
     model: PathBuf,
 
     /// Predicts POS tags.
-    #[clap(long)]
+    #[clap(long, action)]
     predict_tags: bool,
 
     /// Do not segment some character types: {D, R, H, T, K, O, G}.
     /// D: Digit, R: Roman, H: Hiragana, T: Katakana, K: Kanji, O: Other, G: Grapheme cluster.
-    #[clap(long)]
+    #[clap(long, action)]
     wsconst: Vec<WsConst>,
 
     /// Do not normalize input strings before prediction.
-    #[clap(long)]
+    #[clap(long, action)]
     no_norm: bool,
 
     /// Evaluation metric: {char, word}.
     /// char: evaluates each charactor boundary.
     /// word: evaluates each word using Nagata's method.
-    #[clap(long, default_value = "char")]
+    #[clap(long, value_parser, default_value = "char")]
```

Comment:
Is there a reason why `value_parser` attribute is used only here?

The [document](https://github.com/clap-rs/clap/blob/v3.2.1/examples/derive_ref/README.md#arg-attributes) says that `action implies #[clap(value_parser)]`.
If possible, I think it is better to use `action` as in other parts.

Reply:
fixed